certified-connectors/Centrical/apiDefinition.swagger.json

```diff
@@ -0,0 +1,200 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Centrical",
+    "description": "Centrical dynamically brings together real-time performance management, personalized microlearning, augmented coaching, and advanced gamification to personally guide employees and their managers to success.",
```

Comment:
@doronneumann could you please change the description just a little bit? It should be concise and reflect the main value and purpose offered by your connector. E.g. `Send learning and performance events to Centrical platform.`
More information about the description can be found [here](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#write-a-description-for-your-connector).

Reply:
Sure, changes as you suggested.